app-editors/vis/vis-0.4.ebuild

```diff
@@ -1,9 +1,9 @@
 # Copyright 1999-2017 Gentoo Foundation
```

Comment:
Please split version bump into separate commits, one adding the new version and the following one removing the old one, so that it's easier to revert the removal in case there are issues.